### PR TITLE
Update installing.adoc with the final 5.1.1 build number

### PR DESCRIPTION
**Pull request by HanaGemela**
Change client version number in `installing.adoc`

Backport to 5.1 only

### CODE REVIEW

File: site.yml [hunk 1]
```diff
@@ -31,8 +31,8 @@ asciidoc:
     # Antora pagination (prev page, next page)
     page-pagination: true
 #   desktop
-    latest-desktop-version: '5.0'
-    previous-desktop-version: '4.2'
+    latest-desktop-version: '5.1'
+    previous-desktop-version: '5.0'
```
Comment (mmattel) [inline site.yml:35]: This change is ok but does nothing as this is only relevant for local building and will not affect the docs build being published to the web. For your info, this change only **prints** the version in the text where used.

File: antora.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 name: desktop
 title: Desktop App
-version: 'next'
+version: '5.1'
```
Comment (mmattel) [inline antora.yml:3]:
```suggestion
version: 'next'
```
Reverting. Do not fiddle on antora build stuff. If you have questions you are welcomed to ask.

File: .drone.star [hunk 3]
```diff
@@ -6,7 +6,7 @@ def main(ctx):
     # Version shown as latest in generated documentations
     # It's fine that this is out of date in version branches, usually just needs
     # adjustment in master/deployment_branch when a new version is added to site.yml
-    latest_version = "5.0"
+    latest_version = "5.1"
```
Comment (mmattel) [inline .drone.star:9]: This change is ok for now but is only relevant for local building and will not affect the docs build being published to the web. Please ask upfront when you intend to do such changes.

File: modules/ROOT/pages/installing.adoc [hunk 4]
```diff
@@ -1,8 +1,8 @@
 = Installing the Desktop App
 :toc: right
 :toclevels: 4
-:client-version-win: 5.1.0.12434-rc.1.x64
-:client-version-appimage: 5.1.0.12434-rc.1-x86_64
+:client-version-win: 5.1.2.12260.x64
+:client-version-appimage: 5.1.2.12260-x86_64
```
Comment (michaelstingl) [inline modules/ROOT/pages/installing.adoc:5]: @mmattel please bring only this change to master + 5.1 branch. 🙏 

I broke everything else by mixing-up branches… 🤦‍♂️

Comment (mmattel) [reply modules/ROOT/pages/installing.adoc:5]: done, all is correct now, you can approve.

**Review by EParzefall — APPROVED** [review]

**Pull request merged**